oauthenticator/tests/test_globus.py

```diff
@@ -54,14 +54,27 @@ def globus_client(client):
     return client
 
 
+@fixture
+def user():
+    class User:
+        @gen.coroutine
+        def get_auth_state(self):
+            return {'tokens': {}}
+
+        save_auth_state = Mock()
+
+        name = 'wash'
+    return User()
+
+
```

Comment:
What is the used for?

Reply:
The `User` fixture is a Jupyterhub user mock, intended to track when `get_auth_state` and `save_auth_state` were called. It's only used for the one test on token revocation on logout.